### DISCUSSION TITLE
Converting film negatives (contact sheet) to positive

### DISCUSSION THREAD

**Discussion by jelabarre59** [ImageMagick/ImageMagick6#109] [Q&A]
I know this topic had been discussed on the old forum server, DDG still sees the old location, but can't get to the thread now.

I'm trying to make digital "contact sheets" of all our family negatives, but can't find an application that makes a decent conversion to anything even vaguely looking like the proper positive image.  I've seen a couple of scripts from 2013 that were supposed to do this; one is only for MSWindows (maybe it could be extracted and re-worked), the other one gives be some washed blue colored image.  

For that matter, pretty much everything I've tried gives me the washed-out blue images.  I would presume it's because none of the scripts/applications are compensating for the orange color of the negative's substrate.

I have some sample files at https://flic.kr/s/aHsmRzpmeJ (three strips of 126 film negatives, and scans of the individual frames on the negatives, done with a Kodak Scanza).  I'd still be using the Scanza for individual frames, but only after getting the contact sheets to see what's worth scanning (some for family use, and some that seem like good photos for Creative Commons "stock images").  

Additionally, there may be some larger-format negatives I'd scan, but very few of them would be color negatives, and B&W negatives are simple to convert

**Comment by jelabarre59** [comment]
I found that another digital camera (one that the sensor quit working on 9 years ago, and suddenly started working again last month) can be set to take "raw" images.  So I can get those and use them directly or convert to tiff.  But even then the conversions come out looking horrible, washed out blue and un-viewable images.  neg2pos and negative2positive make an attempt, but no matter how I fiddle with the adjustments I can't get anything but washed-out images.

There needs to be a conversion formula, something that compensates for the orange emulsion without turning everything into a washed out blob.

**Comment by urban-warrior** [comment]
@fmw42, do you have any recommendations for @jelabarre59?

**Reply by fmw42** [reply]
He has already tried my two scripts without success. So I have further suggestions. Sorry.

I could not download the one example of 3 negatives. But for my scripts to work, those scans would need to be separate and rotated properly. It is possible that the scan background is causing issues also.
